infrastructure/Dockerfile

```diff
@@ -203,6 +205,9 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+RUN [[ -e "/sys/kernel/mm/ksm/run" ]] \
+    && echo 1 >/sys/kernel/mm/ksm/run \
+    && echo 1000 >/sys/kernel/mm/ksm/sleep_millisecs
```

Comment:
not sure if de-duplication will work during creation of the container
/sys/ is created dynamically - must be set in run time mode